src/main/java/seedu/address/model/person/Person.java

```diff
@@ -116,8 +146,14 @@ public String toString() {
                 .append(getPhone())
                 .append("; Email: ")
                 .append(getEmail())
+                .append("; Parent Phone: ")
+                .append(getParentPhone())
+                .append("; Parent Email: ")
+                .append(getParentEmail())
                 .append("; Address: ")
                 .append(getAddress())
+                .append("; Outstanding Fees: ")
+                .append(getFee())
                 .append("; Remark: ")
                 .append(getRemark());
```

Comment:
for my optional fields, `toString()` only prints the fields when they are not empty. Perhaps you can change it also, so that it seems neater?

e.g.
```
if (!getSchool().isEmpty()) {
        builder.append("; School: ")
                .append(getSchool());
}
```